figures/generate-figures.sh

```diff
@@ -101,3 +101,23 @@ bash ${analyses_dir}/copy_number_consensus_call/run_consensus_call.sh
 # Run CN status heatmap but use parameter so file is saved to figures folder
 Rscript -e "rmarkdown::render('${analyses_dir}/cnv-chrom-plot/cn_status_heatmap.Rmd',
                               clean = TRUE, params = list(final_figure=TRUE))"
+
+							  
+####### Telomerase Activities
+
+
+#enviroment settings
+
+set -e
+set -o pipefail
+
+
+#generate telomerase activities using gene expression data from collapse RNA seq data files
+Rscript --vanilla 01-run-EXTEND.R --input ${analyses_dir}/collapse-rnaseq/results/pbta-gene-expression-rsem-fpkm-collapsed.stranded.rds --output ${analyses_dir}/telomerase-activity-prediction/results/TelomeraseScores_PTBAStranded_FPKM.txt
```

Comment:
It looks like only this file, `TelomeraseScores_PTBAStranded_FPKM.txt` is now used in the script to run the figures. Can we remove the lines that are no longer used to generate files consumed by `scripts/TelomeraseActivitites.R`?

Reply:
Hi Dr. Stephanie
I have addressed all the things you have mentioned. 
Thanks
Nighat